content/frontend/react-apollo/1-getting-started.md

```diff
@@ -34,43 +63,59 @@ yarn create react-app hackernews-react-apollo
 
 </Instruction>
 
-> **Note**: This tutorial uses [Yarn](https://yarnpkg.com/) for dependency management. Find instructions for how you can install it [here](https://yarnpkg.com/en/docs/install). If you prefer using `npm`, you can just run the equivalent commands. 
+> **Note**: This tutorial uses [yarn](https://yarnpkg.com/)
+> for dependency management. Find instructions for how you
+> can install it
+> [here](https://yarnpkg.com/en/docs/install). If you prefer
+> using `npm`, you can just run the equivalent commands.
 
-This will create a new directory called `hackernews-react-apollo` that has all the basic configuration setup.
+This will create a new directory called
+`hackernews-react-apollo` that has all the basic
+configuration setup.
 
-Make sure everything works by navigating into the directory and starting the app:
+Make sure everything works by navigating into the directory
+and starting the app:
 
 ```bash
 cd hackernews-react-apollo
 yarn start
 ```
 
-This will open a browser and navigate to `http://localhost:3000` where the app is running. If everything went well, you'll see the following:
+This will open a browser and navigate to
+`http://localhost:3000` where the app is running. If
+everything went well, we'll see the following:
 
 ![](http://imgur.com/Yujwwi6.png)
 
 <Instruction>
 
-To improve the project structure, move on to create two directories, both inside the `src` folder. The first is called `components` and will hold all our React components. Call the second one `styles`, that one is for all the CSS files you'll use.
+To improve the project structure, create two directories,
+both inside the `src` folder. The first is called
+`components` and will hold all our React components. Create
+a second directory called `styles` to hold all of the CSS
+for the project.
 
-`App.js` is a component, so move it into `components`. `App.css` and `index.css` contain styles, so move them into `styles`. You also need to change the references to these files in both `index.js` and `App.js` accordingly:
+`App.js` is a component, so move it into `components`.
+`App.css` and `index.css` contain styles, so move them into
+`styles`. We also need to change the references to these
+files in both `index.js` and `App.js` accordingly:
 
 </Instruction>
 
 ```js{4}(path=".../hackernews-react-apollo/src/index.js")
-import React from 'react'
-import ReactDOM from 'react-dom'
-import './styles/index.css'
-import App from './components/App'
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './styles/index.css';
+import App from './components/App';
```

Comment:
The imports in CRA have changed, let's include all imports that the reader sees at this point

```suggestion
import App from './components/App';
import reportWebVitals from './reportWebVitals';
```